config/routes.rb

```diff
@@ -69,6 +69,7 @@
   get '/collections/:id/download_xml/:format(/:revision_id)' => 'collections#download_xml', as: 'download_collection_xml'
   get '/collections/:id/create_delete_proposal' => 'collections#create_delete_proposal', as: 'create_delete_proposal_collection'
   get '/collections/:id/create_update_proposal' => 'collections#create_update_proposal', as: 'create_update_proposal_collection'
+  get '/collections/:id/loss' => 'collections#loss_report', as: 'loss_report_collections'
```

Comment:
I think we should make the url `loss_report` as well, that will probably be more user friendly/less scary